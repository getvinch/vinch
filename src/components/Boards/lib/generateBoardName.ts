export default function() {
  return `Board - ${new Date().toLocaleDateString(navigator.language, {
    year: 'numeric',
    month: 'numeric',
    day: 'numeric',
    hour: 'numeric',
    minute: 'numeric',
  })}`;
}
