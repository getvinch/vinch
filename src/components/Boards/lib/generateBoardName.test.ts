import generateBoardName from './generateBoardName';
const MockDate = require('mockdate');

describe('generateBoardName', () => {
  beforeAll(() => {
    MockDate.set('1/30/2000');
  });

  afterAll(() => {
    MockDate.reset();
  });

  it('generates a name based on date', () => {
    const generatedBoardName = generateBoardName();
    expect(generatedBoardName).toContain('1/30/2000');
    expect(generatedBoardName).toMatchSnapshot();
  });
});
