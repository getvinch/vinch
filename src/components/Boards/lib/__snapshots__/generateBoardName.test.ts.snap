// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateBoardName generates a name based on date 1`] = `"Board - 1/30/2000, 12:00 AM"`;
